frontend/public/locales/en/public.json

```diff
@@ -1610,5 +1610,8 @@
   "default": "default",
   "OK": "OK",
   "Disabled": "Disabled",
-  "Enabled": "Enabled"
+  "Enabled": "Enabled",
+  "Pipeline status is {{status}}. View logs.": "Pipeline status is {{status}}. View logs.",
+  "Pipeline not started. Start pipeline.": "Pipeline not started. Start pipeline.",
+  "Pipeline not started": "Pipeline not started"
```

Comment:
These should be in the Pipeline package (see my earlier comment)

Reply:
Solved.